fairing/deployers/gcp/gcpserving.py

```diff
@@ -5,27 +5,31 @@
 from googleapiclient import discovery
 from googleapiclient import errors
 
-# TODO: Integrate with DeployerInterface
-class GCPServingDeployer:
+# TODO: Implement predict and delete methods.
```

Comment:
Should predict and delete be methods on the DeployerInterface? Training jobs also use this interface.
Maybe

TrainingInterface, PredictionInterface -> DeployerInterface

SeldonPrediction -> PredictionInterface -> DeployerInterface
GCPPrediction -> PredictionInterface -> DeployerInterface

Reply:
I wasn't referring to having those methods be part of the interface in this comment, but that hierarchy seems reasonable to me.